cfgov/unprocessed/apps/owning-a-home/js/explore-rates/dom-values.js

```diff
@@ -0,0 +1,38 @@
+const unFormatUSD = require( '../../node_modules/unformat-usd' );
+
+/**
+ * Get value(s) of an individual HTML element in the control panel.
+ * @param   {string} param Name of parameter to get.
+ *                         Usually the HTML element's id attribute.
+ * @returns {Object} Hash of element id and its value(s).
+ */
+function getSelection( param ) {
+
+  const elm = document.querySelector( '#' + param );
+  let val;
+
+  if ( !elm ) {
+    return val;
```

Comment:
Why return an `undefined` variable and not just `false`?

Reply:
This used to tie into the [params object](https://github.com/cfpb/owning-a-home/blob/master/src/static/js/modules/explore-rates.js#L73) and use https://api.jquery.com/jquery.extend/, which throws out undefined and null values. So I kept that functionality.

Reply:
Ah gotcha, thanks for the explanation.